src/interfaces/api-filing-interface.ts

```diff
@@ -65,7 +65,7 @@ export interface ApiFilingIF {
       toLegalType?: CorpTypeCd
     }
 
-    amalgamation?: any // some object
+    amalgamationApplication?: any // some object
```

Comment:
In a future PR, please type this :)

Also in api-task-interface.ts below.

Reply:
Fixed, thanks.

Reply:
I might have to create an interface in the shared components repo for this. I see that most of them are any and the ones that are not, are in the shared repo. I can take of this in the next PR!

Ref:
https://github.com/bcgov/business-filings-ui/blob/f607203812b44da61395e0e7c777c33377ec7158/src/interfaces/api-task-interface.ts#L41